Cartfile

```diff
@@ -10,4 +10,5 @@ github "stephencelis/SQLite.swift" == 0.12.2
 github "NYPL-Simplified/r2-shared-swift" "develop"
 github "NYPL-Simplified/r2-streamer-swift" "develop"
 github "readium/r2-navigator-swift" "develop"
-github "NYPL-Simplified/audiobook-ios-overdrive" "master"
+github "NYPL-Simplified/r2-lcp-swift" "develop"
```

Comment:
can you explain why do we need a fork? I took a quick look at the fork and it seemed identical to the one from readium.

Reply:
we don't need this fork anymore. I added changes to r2-lcp-swift to support separate license files .lcpl at first (when we didn't have a test library and it wasn't clear how that all should work) but later removed all the changes as the library embeds the license file into the book as a part of fulfilment process.

Reply:
switched to the readium version

Reply:
actually I think we need this fork, without it Carthage starts fetching r2-shared-swift infinitely.